sys/sys/thr.h

```diff
@@ -55,6 +55,7 @@ struct thr_param {
     long	*parent_tid;		/* parent accesses the new TID here. */
     int		flags;			/* thread flags. */
     struct rtprio	*rtp;		/* Real-time scheduling priority */
+    /* XXXAR: should ddc always be here? It is only used by CHERIABI. */
```

Comment:
The size of `struct thr_param` is part of the syscall ABI.  Changes for CheriABI are ok since that ABI isn't fixed (and won't be for quite a while), but we can't increase it without adding compatibility shims to the kernel.